movie_api/core/views.py

```diff
@@ -1,16 +1,24 @@
 from django.shortcuts import render
 from django.http import HttpResponse
+from django.http import JsonResponse
 
 import os
 import logging
 
+log = logging.getLogger(__name__)
+
 from movie_client import MovieClient
 
 if "API_KEY" in os.environ:
     movie_client = MovieClient(os.environ['API_KEY'])
```

Comment:
you could move this conditional into the movie_client constructor if you want but I'm not opposed to it at this stage 👍